### ISSUE TITLE
Package is outdated on packages.firegento.com and satis.yireo.com

### ISSUE THREAD

**Issue by aadmathijssen** [yireo/Yireo_EmailOverride#28]
Hi,

Both packages.firegento.com and satis.yireo.com report that the latest version of the Yireo_EmailOverride module is 0.2.13, while currently version 0.2.16 is out since 25-10-2016.

Can you make sure these Satis repositories are updated?

Thanks.

Aad

**Comment by jissereitsma** [comment]
Thanks for the ping. Our Satis is updating itself via cron, however this cronjob stopped working because of a mismatch between the local satis server and the server-wide composer install. This is now being fixed. And our own Satis again bubbles to the Firegento Satis as well, so this should be fixed within a few days. Again, thanks for reporting this!

**Comment by aadmathijssen** [comment]
I used packages.firegento.com as my main repository, but it didn't seem to get updated. Even after 12 days, still no change. Turns out that I was using the Composer package connect20/yireo_emailoverride package instead of yireo/yireo_emailoverride. After switching to that package, I was able to upgrade to the latest version.

As far as I'm concerned, this issue can be closed.

Thanks for the quick reply in the first place!
